### PR TITLE
Integrate jaxb-ri 2.3.3-b02, jaxws-ri 2.3.3-b01, jaxrpc-ri 1.1.6

### PR DESCRIPTION
**Pull request by lukasj**
Avoid (some) split packages with jaxws-ri
Update build plugins
Remove obsolete code

Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

File: wsit/bundles/webservices-osgi-aix/pom.xml [hunk 1]
```diff
@@ -290,19 +291,28 @@
...
+                        <dependency>
+                            <groupId>org.glassfish.grizzly</groupId>
+                            <artifactId>grizzly-portunif</artifactId>
+                            <version>2.4.4</version>
+                        </dependency>
+                        <dependency>
+                            <groupId>org.apache.tomcat</groupId>
+                            <artifactId>tomcat-coyote</artifactId>
+                            <version>9.0.8</version>
+                        </dependency>
```
Comment (Verdent) [inline wsit/bundles/webservices-osgi-aix/pom.xml:315]: Why not to place these dependencies to some common parent pom and reference them just from here? I saw that you have added the same dependencies in webservices-osgi module also.

File: wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/ws/security/trust/impl/IssueSamlTokenContractImpl.java [hunk 2]
```diff
@@ -175,7 +175,7 @@ private EncryptedKey encryptKey(final Document doc, final byte[] encryptedKey, f
             cipher.init(XMLCipher.WRAP_MODE, pubKey);
 
             encKey = cipher.encryptKey(doc, new SecretKeySpec(encryptedKey, "AES"));
-            final KeyInfo keyinfo = new KeyInfo(doc);
+            final KeyInfo keyinfo = new KeyInfo(doc);//tady
```
Comment (Verdent) [inline wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/ws/security/trust/impl/IssueSamlTokenContractImpl.java]: Did you forget to remove this comment or is it intentional?

**Review by Verdent — CHANGES_REQUESTED** [review]

File: wsit/bundles/webservices-osgi-aix/pom.xml (hunk 1 above)
Comment (lukasj) [reply wsit/bundles/webservices-osgi-aix/pom.xml:315]: that's going to happen in the next set of changes

File: wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/ws/security/trust/impl/IssueSamlTokenContractImpl.java (hunk 2 above)
Comment (lukasj) [reply wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/ws/security/trust/impl/IssueSamlTokenContractImpl.java]: whoops :-), fixed

**Review by Verdent — APPROVED** [review]
LGTM

**Pull request merged**